### PR TITLE
Abhishek Added algorithm

### PR DESCRIPTION
**Pull request by AbhishekSingh1853**

### CODE REVIEW

**Review by gavand-sagar — APPROVED** [review]
looks good

**Review by Ak81090 — APPROVED** [review]
looks good.

**Comment by kathiresan-g** [comment]
HI BRO

**Review by 319umangi — APPROVED** [review]

**Review by maheshkh93 — APPROVED** [review]
good

**Review by Ak81090 — APPROVED** [review]
nice..

**Review by PAGAR-2022 — APPROVED** [review]
good

File: Abhishek/Algoritham.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+display greeting
+Step 1 start
+step 2 " Welcome "
+Step 3" display Welcome"
```
Comment (GirjashankarJ) [inline Abhishek/Algoritham.txt:4]: It looks good

**Review by GirjashankarJ — APPROVED** [review]

**Review by Hemant2266 — APPROVED** [review]

**Review by Hemant2266 — APPROVED** [review]

**Review by Hemant2266 — APPROVED** [review]

**Review by goldi2401 — APPROVED** [review]

**Review by Mansi2810 — APPROVED** [review]

**Review by Mohit-Pal — COMMENTED** [review]
where i am in process

**Pull request merged**